packages/react-components/theme-designer/src/components/Header/Header.stories.tsx

```diff
@@ -0,0 +1,3 @@
+import { Header } from './Header';
+export default { component: Header };
+export const Default = {};
```

Comment:
I see you deleted the other stories for the components, is this one needed?

Reply:
rem. thanks!

Reply:
rem. thanks !